src/main/java/io/github/townyadvanced/townyprovinces/settings/ConfigNodes.java

```diff
@@ -168,7 +168,100 @@ public enum ConfigNodes {
+		"",
+		"############################################################",
+		"# +------------------------------------------------------+ #",
+		"# |                MAP PLUGIN INTEGRATION                | #",
+		"# +------------------------------------------------------+ #",
+		"############################################################",
+		""),
+	TOWN_COSTS_ICON(
+		"map_integration.town_costs_icon",
+		"",
+		""),
+	TOWN_COSTS_ICON_URL(
+		"map_integration.town_costs_icon.url",
+		"https://cdn-icons-png.flaticon.com/512/9729/9729309.png",
+		"",
+		"# Icon for the town costs. This must be a valid image URL.",
+		"# Default coin icon created by Md Tanvirul Haque - Flaticon",
+		"# https://www.flaticon.com/free-icon/dollar_9729309"),
+	TOWN_COSTS_ICON_HEIGHT(
+		"map_integration.town_costs_icon.height",
+		"35",
+		"",
+		"# Height in pixels for the town costs icon to be displayed as."),
+	TOWN_COSTS_ICON_WIDTH(
+		"map_integration.town_costs_icon.width",
+		"35",
+		"",
+		"# Width in pixels for the town costs icon to be displayed as."),
+	DYNMAP(
+		"map_integration.dynmap",
+		"",
+		"",
+		"",
+		"# +------------------------------------------------------+ #",
+		"# |                        DYNMAP                        | #",
+		"# +------------------------------------------------------+ #",
+		""),
+	DYNMAP_USE_TOWN_COSTS_ICON(
+		"map_integration.dynmap.use_town_costs_icon",
+		"false",
+		"",
+		"# Toggle whether Dynmap should use your specified town_costs_icon, or an inbuilt Dynmap icon."),
+	PL3XMAP(
+		"map_integration.pl3xmap",
+		"",
+		"",
+		"",
+		"# +------------------------------------------------------+ #",
+		"# |                        PL3XMAP                       | #",
+		"# +------------------------------------------------------+ #",
+		""),
+	PL3XMAP_PROVINCES_LAYER(
+		"map_integration.pl3xmap.provinces_layer",
+		"",
+		""),
+	PL3XMAP_PROVINCES_LAYER_PRIORITY(
+		"map_integration.pl3xmap.provinces_layer.priority",
+		"6",
+		"",
+		"# You do not need to change this unless other map addons conflict with the provinces layer."),
+	PL3XMAP_PROVINCES_LAYER_ZINDEX(
+		"map_integration.pl3xmap.provinces_layer.zindex",
+		"250",
+		"",
+		"# You can decrease this value to blend the provinces layer with the map more."),
+	PL3XMAP_PROVINCES_LAYER_TOGGLEABLE(
+		"map_integration.pl3xmap.provinces_layer.toggleable",
+		"true",
+		"",
+		"# Set to false to disallow users from toggling the provinces layer."),
+	PL3XMAP_TOWN_COSTS_LAYER(
+		"map_integration.pl3xmap.town_costs_layer",
+		"",
+		""),
+	PL3XMAP_TOWN_COSTS_LAYER_PRIORITY(
+		"map_integration.pl3xmap.town_costs_layer.priority",
+		"6",
+		"",
+		"# You do not need to change this unless other map addons conflict with the town costs layer."),
+	PL3XMAP_TOWN_COSTS_LAYER_ZINDEX(
+		"map_integration.pl3xmap.town_costs_layer.zindex",
+		"250",
+		"",
+		"# You can decrease this value to blend the town costs layer with the map more."),
+	PL3XMAP_TOWN_COSTS_LAYER_TOGGLEABLE(
```

Comment:
❓ 
- Question: Will the town costs layer be "toggled off" by default
- If possible we want that, both to be consistent with dynmap. and so that players don't see that usually non-required visual layer unless they ask for it.

Reply:
Town Costs Layer is toggled off by default to match Dynmap

Toggleable config node controls whether the ability to toggle the layer is given to the end user 

I will remove this config node for Town Costs layer, without the ability to toggle it _on_ the layer is useless so this should probably not be something that can be disabled

I will keep this for Provinces layer as servers may prefer not having the ability to disable it for aesthetic reasons

Reply:
I'm a little confused.

So question:

With the current PR code, is it correct to say that when the plugin installs with all default configs:
1. When the user access the plexmap, the borders layer is on, but can be toggle off by the user, and 
2. When the user access the plexmap, the costs layer is off, but can be toggle on by the user.

?

Reply:
Yes, that's correct for default config!

The config node `PL3XMAP_PROVINCES_LAYER_TOGGLEABLE` allows the server to prevent toggling, forcing the Borders layer to be always-on, without the ability for the user to disable it

The config node `PL3XMAP_TOWN_COSTS_LAYER_TOGGLEABLE` has been removed, it would have allowed the server to prevent toggling, forcing the Town Costs layer to be always-off, without the ability to enable

Default:
- Borders: On - **Can** be toggled _off_
- Town Costs: Off - **Can** be toggled _on_

Configs set to false:
- Borders: On - **Cannot** be toggled _off_
- REMOVED however _would be_ Town Costs: Off - **Cannot** be toggled _on_

Hopefully this is clearer, the config option simply decides whether the user can see layer controls, it does not affect function outside of this